tools/base/fpgainfo/bmcdata.c

```diff
@@ -63,26 +64,26 @@ fpga_result get_metrics(fpga_token token,
         uint64_t j = 0;
         switch (inquiry) {
         case ALL:
-            for (i = 0; i < *num_metrics; ++i) {
+            for (i = 0; i < *num_metrics_info; ++i) {
                 id_array[j++] = i;
             }
             break;
         case POWER:
-            for (i = 0; i < *num_metrics; ++i) {
+            for (i = 0; i < *num_metrics_info; ++i) {
                 if (metrics_info[i].metric_type == FPGA_METRIC_TYPE_POWER) {
                     id_array[j++] = i;
                 }
             }
             break;
         case THERMAL:
-            for (i = 0; i < *num_metrics; ++i) {
+            for (i = 0; i < *num_metrics_info; ++i) {
                 if (metrics_info[i].metric_type == FPGA_METRIC_TYPE_THERMAL) {
                     id_array[j++] = i;
                 }
             }
             break;
         case PERF:
```

Comment:
PERF is too generic a name.  We should add a prefix.

Reply:
fixed